packages/debug/src/browser/debug-session-manager.ts

```diff
@@ -214,7 +214,7 @@ export class DebugSessionManager {
                     }
                 }
 
-                const sessionId = await this.debug.createDebugSession(resolved.configuration);
+                const sessionId = await this.debug.createDebugSession(resolved.configuration, options.workspaceFolderUri);
```

Comment:
It does not appear that the implementation of the `createDebugSession` method on the `DebugServiceImpl` has been updated to handle the `workspaceFolderUri`. Is that intentional?

Reply:
Thank you. No, this was an oversight, since this value is not needed there.
In the plain Theia API the ws folder is taken from the `DebugSessionOptions` and we only need it for vs code compatibility.
I've added the parameter as `_workspaceFolderUri?`